packages/codegen/src/config.test.ts

```diff
@@ -0,0 +1,47 @@
+import { Config, defaultConfig, FieldConfig, isFieldIgnored } from "./config";
+import { makeEntity } from "./EntityDbMetadata";
+
+describe("config", () => {
+  describe("isFieldIgnored", () => {
+    // A little hack to test the case of `ignore: false`, since it is actually defined as `ignore?: true`
+    const falseAsTrue = (false as boolean) as true;
+
+    it.each([
+      ["fields key is omitted all together", newAuthorConfig()],
+      ["fields does not specify the field in question", newAuthorConfig({})],
+      ["field is specified but doesn't have ignore key", newAuthorConfig({ shouldNotIgnore: {} })],
```

Comment:
Fwiw this `shouldNotIgnore` name threw me off just a bit, it almost looks like a config option itself, instead of the (what I finally realized) made-up-field name we used for the test. Maybe using something that is "very obviously a column/field name" like `firstName` would make it more apparent to the reads what this is.

Also "shouldNotIgnore" the name is like mixing the "then" / expected behavior into the code-/data-under-test itself...